src/dso_api/dynamic_api/remote/views.py

```diff
@@ -183,6 +176,18 @@ def _call_remote(self, url="", query_params={}) -> Union[dict, list]:  # noqa: C
 
         return self._raise_http_error(response)
 
+    def _make_url(self, path: str, query_params: dict) -> str:
```

Comment:
`URL` is-a `str` so this is correct. But if you change the return type to `URL` you can continue building up URLs that `_make_url` returns